lib/fog/aws/models/rds/cluster.rb

```diff
@@ -0,0 +1,93 @@
+module Fog
+  module AWS
+    class RDS
+      class Cluster < Fog::Model
+        identity :id, :aliases => 'DBClusterIdentifier'
+
+        attribute :allocated_storage,            :aliases => 'AllocatedStorage',          :type => :integer
+        attribute :backup_retention_period,      :aliases => 'BackupRetentionPeriod',     :type => :integer
+        attribute :db_cluster_members,           :aliases => 'DBClusterMembers',          :type => :array
+        attribute :db_cluster_parameter_group,   :aliases => 'DBClusterParameterGroup'
+        attribute :db_subnet_group,              :aliases => 'DBSubnetGroup'
+        attribute :endpoint,                     :aliases => 'Endpoint'
+        attribute :engine,                       :aliases => 'Engine'
+        attribute :engine_version,               :aliases => 'EngineVersion'
+        attribute :password,                     :aliases => 'MasterUserPassword'
+        attribute :master_username,              :aliases => 'MasterUsername'
+        attribute :port,                         :aliases => 'Port',                      :type => :integer
+        attribute :preferred_backup_window,      :aliases => 'PreferredBackupWindow'
+        attribute :preferred_maintenance_window, :aliases => 'PreferredMaintenanceWindow'
+        attribute :state,                        :aliases => 'Status'
+        attribute :vpc_security_groups,          :aliases => 'VpcSecurityGroups'
+
+        attr_accessor :storage_encrypted #not in the response
+
+        def ready?
+          state == "available"
+        end
+
+        def snapshots
+          requires :id
+          service.cluster_snapshots(:cluster => self)
+        end
+
+        def servers(set=db_cluster_members)
+          set.map do |member|
+            service.servers.get(member['DBInstanceIdentifier'])
+          end
+        end
+
+        def master
+          db_cluster_members.detect { |member| member["master"] }
```

Comment:
unfortunate thing about the `master` and `replicas` methods is that you have inflate the entire list (1 call per member) and then filter.  not a blocker but definitely room for a performance enhancement.